components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -382,8 +387,28 @@ protected void initiateAuthenticationRequest(HttpServletRequest request, HttpSer
                                                  AuthenticationContext context) throws AuthenticationFailedException {
 
         try {
+            if (LoggerUtils.isDiagnosticLogsEnabled()) {
+                DiagnosticLog.DiagnosticLogBuilder diagnosticLogBuilder = new DiagnosticLog.DiagnosticLogBuilder(
+                        getComponentId(), VALIDATE_OUTBOUND_AUTH_REQUEST);
```

Comment:
let's change the action ID to something that denotes the purpose of the method more accurately

Reply:
something like INITIATE_AUTHENTICATION_REQUEST

Reply:
Fixed with 2f3dbb21b41f2c19722078c1b25bd02d48cc434f